lib/api/remoteActions/index.js

```diff
@@ -54,7 +55,10 @@ module.exports = function RemoteActions (kuzzle) {
     }
 
     this.kuzzle.config = config(params);
-    this.kuzzle.services.init({blacklist: ['mqBroker', 'logger', 'notificationCache']});
+    // the only service we need is the internal broker
+    this.kuzzle.services.list = {
+      broker: new InternalBroker(kuzzle, {})
+    };
     this.kuzzle.pluginsManager = new PluginsManager(this.kuzzle);
     this.kuzzle.pluginsManager.init(this.kuzzle.isServer, true);
```

Comment:
I think this is my fault: in one of my previous PR, I changed the `pluginsManager.init` method to make it return a Promise, because it now needs to fetch plugins configuration from the database.
So `pluginsManager.run()` should only be started after `pluginsManager.init()` promise resolves.

Sorry about that :-(